polaris/ocean/tests/single_column/ideal_age/__init__.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from polaris import TestCase
+from polaris.ocean.tests.single_column.forward import Forward
+from polaris.ocean.tests.single_column.initial_state import InitialState
+from polaris.ocean.tests.single_column.viz import Viz
+from polaris.validate import compare_variables
+
+
+class IdealAge(TestCase):
+    """
+    The default test case for the single column test group simply creates
+    the mesh and initial condition, then performs a short forward run on 4
+    cores.
+    """
+    def __init__(self, test_group, resolution, ideal_age=True):
+        """
+        Create the test case
+        Parameters
+        ----------
+        test_group : polaris.ocean.tests.single_column.SingleColumn
+            The test group that this test case belongs to
+        """
+        name = 'ideal_age'
+        self.resolution = resolution
+        if resolution >= 1.:
+            res_str = f'{resolution:g}km'
+        else:
+            res_str = f'{resolution * 1000.:g}m'
+        subdir = os.path.join(res_str, name)
+        super().__init__(test_group=test_group, name=name,
+                         subdir=subdir)
+        self.add_step(
+            InitialState(test_case=self, resolution=resolution,
+                         ideal_age=ideal_age))
+
+        self.add_step(
+            Forward(test_case=self, ntasks=1, min_tasks=1,
+                    openmp_threads=1, ideal_age=ideal_age))
```

Comment:
This is my suggestion so we don't need to change the `forward.py` at all in the test group.

```suggestion
        step = Forward(test_case=self, ntasks=1, min_tasks=1,
                       openmp_threads=1)
        step.add_yaml_file('polaris.ocean.tests.single_column.ideal_age',
                               'forward.yaml')
        self.add_step(step)
  ```

Reply:
added